universidade-estadual-de-alagoas-abnt.csl

```diff
@@ -6,16 +6,16 @@
     <id>http://www.zotero.org/styles/universidade-estadual-de-alagoas-abnt</id>
     <link href="http://www.zotero.org/styles/universidade-estadual-de-alagoas-abnt" rel="self"/>
     <link href="http://www.zotero.org/styles/associacao-brasileira-de-normas-tecnicas" rel="template"/>
-    <link href="http://www.uneal.edu.br/pro-reitorias/prograd/regulamento-do-tcc-uneal.pdf" rel="documentation"/>
+    <link href="https://www.abntcatalogo.com.br/norma.aspx?ID=408006" rel="documentation"/>
```

Comment:
Why the change here? My sense is that there are different interpretations of the ABNT (that's why we have multiple versions and why yours is named "universidade-estadual-de-alagoas" -- wouldn't it make more sense to link to your specific guide?

Reply:
> Why the change here? My sense is that there are different interpretations of the ABNT (that's why we have multiple versions and why yours is named "universidade-estadual-de-alagoas" -- wouldn't it make more sense to link to your specific guide?

I understand your doubt, but I do not believe that they are different interpretations, but the different alternatives that ABNT offers, or particularities adopted by the institutions that offer their styles. For example, we chose to insert the DOI, which not all use in bibliographic references; we chose not to use abbreviations in the names of the authors. These are some of the examples.

Reply:
Right, that makes sense, but wouldn't it then make sense to link to your description since that presumably contains which options you chose?

Reply:
> Right, that makes sense, but wouldn't it then make sense to link to your description since that presumably contains which options you chose?

Right. I think I get it. Do you have any suggestions? I don't think line 9 might exist.

Reply:
I think line 9 might not exist.

Reply:
Not sure I understand "might not exist" in this context. Is it OK to revert this to the previous guide (which is still available online)?

Reply:
> Not sure I understand "might not exist" in this context. Is it OK to revert this to the previous guide (which is still available online)?

No problem.